js/index.js

```diff
@@ -161,11 +137,82 @@ contactBody[2].innerHTML = siteContent["contact"]["email"]
 let footer = document.querySelector("footer")
 footer.innerHTML = siteContent["footer"]["copyright"];
 
+// Start DOM - II 
+// 10 unique event-listeners 
+
+let ctaImg = document.querySelector(".cta img")
+let ctaTxt = document.querySelector(".cta h1")
+let ctaDiv = document.querySelector(".cta-text")
+let ctaDivM = document.querySelector(".cta")
+let navBar = document.querySelectorAll("a")
+
+document.addEventListener('copy', () => {
+  alert("GOT YOU! STOP STEALING MY SHIT!")
+})
+
```

Comment:
let's change the alert text to something else. Employers might read your code. 
👍